Engine/Handlers/Script.cpp

```diff
@@ -23,6 +23,8 @@
 #include <algorithm>
 #include <vector>
 #include <unordered_map>
+#include<limits>
```

Comment:
Why is this needed? And please fix the indentation.

Reply:
numeric limits isn't included without it